src/Helper.php

```diff
@@ -100,6 +101,12 @@ public function canBypassRestriction(AccountInterface $account = NULL) {
    */
   public function hasEntitySitesAccess(EntityInterface $entity, array $user_sites) {
     if (empty($user_sites)) {
+      // Get the current user roles.
+      $user_roles = ($this->currentUser->id()) ? User::load($this->currentUser->id())->getRoles() : '';
+      if (!empty($user_roles)) {
+        // Administrator role can bypass the restriction.
+        return (in_array('administrator', $user_roles));
```

Comment:
Should this also be `site_admin`?

Reply:
@anthony-malkoun nope, not needed for site_admin
![Screen Shot 2022-08-04 at 10 16 21 am](https://user-images.githubusercontent.com/20810541/182736674-318e0869-a4bf-454f-8423-c5d5796ed06a.png)